### ISSUE TITLE
AttributeError when clicking something, anything! (on Linux Mint 19.3)

### ISSUE THREAD

**Issue by marc-invalid** [marc-invalid/chipwhisperer-marc#2]
When using the install instructions for Linux Mint 18.x, but actually doing it on a **Linux Mint 19.3** system, the software does not work.  It's not possible to open a project (or even close the window), without getting an error dialog and ```AttributeError: 'NoneType' object has no attribute 'setFirstItemColumnSpanned'```.

This problem is triggered by one of the dependencies and can be worked around by downgrading it to an older version:

```
#--- downgrade pyqtgraph to v0.10.0

pip uninstall pyqtgraph
apt-get install python-pyqtgraph

#--- fix some other warnings

apt-get install libpng-dev python-hidapi python-pyscard
pip install umysql ftd2xx hid visa picoscope
```

**Comment by danyhm** [comment]
same goes for numpy, a downgrade to version 1.16.1 is needed
